src/channel.rs

```diff
@@ -404,7 +405,7 @@ impl ChannelBuilder {
                     // On most modern compiler and architect, c_int is the same as i32,
                     // panic directly to simplify signature.
                     assert!(
-                        val <= i32::from(libc::INT_MAX) && val >= i32::from(libc::INT_MIN),
+                        val <= libc::INT_MAX && val >= libc::INT_MIN,
```

Comment:
I think it's intended to convert as explained at L405.

Reply:
I see. On some platforms they might be different and that will error out. Will just allow the clippy lint there then.